ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -976,7 +976,7 @@ export const AdminInstitutionEdit = fp.flow(
                       </div>
                     )
                   }
-                  disable={this.isAddInstitutionMode}
```

Comment:
bug!  this was not being set correctly.  I'm so glad we will now catch errors like this.

Reply:
since the behavior has been that this is *not* disabled for a long time, I'm removing it instead of correcting it.

Reply:
What happened that we catch errors like this? Is it because you added more typing?

Reply:
Yes, I think so.  This is not flagged as an error on `main`, and my guess is that it's because this component takes `any` as props.  This makes me think we should be on the lookout for other such situations.